openmmapi/include/OpenMM.h

```diff
@@ -61,6 +61,7 @@
 #include "openmm/MonteCarloBarostat.h"
 #include "openmm/MonteCarloMembraneBarostat.h"
 #include "openmm/NonbondedForce.h"
+#include "openmm/NoseHooverChain.h"
```

Comment:
It looks to me like NoseHooverChain is an internal class?  It doesn't appear anywhere in NoseHooverIntegrator's public API.  If so, let's move the header into `include/openmm/internal` and not include it here.